tensorflow/compiler/xla/service/gpu/cudnn_fused_conv_rewriter.cc

```diff
@@ -633,6 +639,96 @@ StatusOr<bool> FuseRelu(HloComputation* comp) {
   return changed;
 }
 
+StatusOr<bool> FuseRelu6(HloComputation* comp, se::CudaComputeCapability cc) {
```

Comment:
The elu code bails out if there is a side-input, saying this is not well supported by cudnn runtime fusion.

Should we be bailing out in relu6 and leaky-relu if there is a side-input?

Reply:
It does seem that bailing out when there's a side-input is necessary, otherwise we generate pathologically-slow relu6's.  I'm testing now to see if this removes *all* of the pathological relu6's that I was seeing.